client/Dockerfile-client

```diff
@@ -0,0 +1,40 @@
+# Our First stage, that builds the application
+FROM node:12.13.0-alpine AS react-builder
+WORKDIR app
+
+# Copy all files 
+COPY . .
+
+# Install dependencies
+RUN yarn
+
+# set graphql server base url
+ARG REACT_APP_GRAPHQL_BASE_URL
+ENV REACT_APP_GRAPHQL_BASE_URL=$REACT_APP_GRAPHQL_BASE_URL
+
+# set ssr server port
+ARG REACT_APP_SSR_PORT
+ENV REACT_APP_SSR_PORT=$REACT_APP_SSR_PORT
+
+# Build application
+RUN yarn build
+
+
+# Our Second stage, that creates an image for production
+FROM node:12.13.0-alpine AS react-prod
+WORKDIR /app
+
+# Copy build folder from stage 1
+COPY --from=react-builder ./app/build ./build
```

Comment:
Please use absolute paths (`COPY --from=react-builder /app/build /app/build`) for clarity.